pytestlx

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```

Comment:
Is this file meant to be included? I am guessing its part of the pipeline testing on Linux, but can we include these values as pipeline variables rather than in the repo?

Reply:
No it wasn't - it was just a local file that I was using to test in WSL. Should be removed by now